l2lsp/identify_method.go

```diff
@@ -0,0 +1,32 @@
+package l2lsp
+
+import (
+	"github.com/HexmosTech/lama2/l2lsp/lsp_req"
```

Comment:
since `l2lsp` is the package name, you don't need to name the file `lsp_req`; simplify to `request` and `response`

Reply:
i had named it as such there were some other things interfering with the name request and response.

Reply:
Actually, you're already within the `l2lsp` package; so you could simply use the name `request`

Reply:
Renamed